packaging/windows-exe/rabbitmq_nsi.in

```diff
@@ -131,7 +131,12 @@ Section "RabbitMQ Service" RabbitService
   ExecDos::exec /DETAILED '"$INSTDIR\rabbitmq_server-%%VERSION%%\sbin\rabbitmq-service.bat" start' ""
   ReadEnvStr $1 "HOMEDRIVE"
   ReadEnvStr $2 "HOMEPATH"
-  CopyFiles "$WINDIR\.erlang.cookie" "$1$2\.erlang.cookie"
+  IfFileExists "$SystemRoot\System32\config\systemprofile\.erlang.cookie" 0 +5
+    CopyFiles "$SystemRoot\System32\config\systemprofile\.erlang.cookie" "$1$2\.erlang.cookie"
```

Comment:
I wonder, if `.erlang.cookie` already exists from a previous installation, will we get an access error due to permissions? The installer should be running with an admin account but it would be worth testing. I have seen strange "access denied" errors when copying cookies that have required deleting the previous one before copying the new one :man_shrugging: